### DISCUSSION TITLE
Change Gallery Default Image Per Page Quantity?

### DISCUSSION THREAD

**Discussion by Prolly92** [stashapp/stash#2255] [Q&A] [answered]
Wondering if there's already a way to change the default image per page quantity? Most of my galleries have more than 40 pictures, I'd like to be able to have them all available on one single page whenever I enter their gallery. I've messed around a bit, but haven't managed to find a way to change the default 40 to any other value.

**Comment by philpw99** [comment, marked as answer]
I think when you reach the end of the 40, the gallery will automatically load the next 40 pictures.

**Reply by Prolly92** [reply]
It will, just takes a bit to load up the next 40 images. I'd just like to figure out how/be able to view everything so I can pick where I want to start, remove the extra clicks to change the quantity every time I go into a gallery with more than 40 images.

**Reply by philpw99** [reply]
Funny, I don't want to promote my app too much, but in this case my Stash Helper can actually help you. It can create unlimited number of images in a temporary gallery. It will warn you if you are viewing more than 1000 pictures, though.
But of course you need to use Windows for my app.
https://github.com/philpw99/Stash_Helper/releases
